pawn/Entities/Players/PlayerSyncedData.pwn

```diff
@@ -112,8 +112,14 @@ class PlayerSyncedData <playerId (MAX_PLAYERS)> {
             case PREFERRED_RADIO_CHANNEL:
                 format(m_preferredRadioChannel, sizeof(m_preferredRadioChannel), "%s", stringValue);
 
-            case VEHICLE_KEYS:
+            case VEHICLE_KEYS: {
                 m_vehicleKeys = intValue;
+                if(m_vehicleKeys | VEHICLE_KEYS_BLINKER_RIGHT) {
```

Comment:
Good catch :) Is this the right check tho? Any number OR'd with 64 will be non-zero, and therefore true.

I think you want:

```javascript
if ((m_vehicleKeys & VEHICLE_KEYS_BLINKER_RIGHT) == 0) {
    // …
    StopBlinking(playerId);
}
```

Reply:
You're totally right!
I don't often have to use bitwise operators. :(